frontend/public/locales/en/translation.json

```diff
@@ -3014,6 +3022,7 @@
   "tooltip.creation.app.reconcileRate": "The frequency of resource reconciliation that is used as a global repository setting. The medium default setting checks for changes to apply every three minutes and re-applies all resources every 15 minutes, even without a change. Select low to reconcile every hour. Select high to reconcile every two minutes. If you select off, the deployed resources are not automatically reconciled.",
   "tooltip.creation.app.settings.existingRule": "If available in the application namespace, you can select a predefined placement configuration",
   "tooltip.creation.app.subReconcileRate": "Turn the auto-reconciliation off for this specific application regardless of the reconcile rate setting in the repository.",
+  "tooltip.creation.nodepool.additional.network": "Specify additional networks to be attached to the nodes in the format '[namespace]/[name]'.",
```

Comment:
This message uses different syntax from the others (`'[namespace]/[name]'` vs `<namespace>/<name>` vs `"[namespace]/[name'"`). Please check if we've used syntax like this before and then be consistent with whatever you find.

Reply:
These are more consistent now, using angle brackets everywhere, but sometimes it is surrounded by quotes, and other times not.